Dockerfile

```diff
@@ -1,18 +1,14 @@
 FROM node:20-alpine3.20 AS build
```

Comment:
@foodelevator Does this dockerfile work or we need some special sauce?

Reply:
how de fak shud i knaw if it worken?

Reply:
I mean it workeding for me on my machine so fak it let's YOLO! :smiley:

Reply:
but nothing special is needed, so stuff should just work